components/server/src/ome/logic/AdminImpl.java

```diff
@@ -634,13 +640,34 @@ public long createSystemUser(Experimenter newSystemUser) {
                 groupProxy(sec.getSecurityRoles().getUserGroupName()));
     }
 
+    @Override
+    @RolesAllowed("system")
+    @Transactional(readOnly = false)
+    public long createLightSystemUser(Experimenter newSystemUser, List<AdminPrivilege> privileges) {
+        assertKnownPrivileges(privileges);
+        final long newSystemUserId = createSystemUser(newSystemUser);
+        newSystemUser = iQuery.get(Experimenter.class, newSystemUserId);
+        final Set<AdminPrivilege> restrictions = new HashSet<>(adminPrivileges.getAllPrivileges());
+        restrictions.removeAll(privileges);
+        final List<NamedValue> userConfig = new ArrayList<NamedValue>(restrictions.size());
+        for (final AdminPrivilege restriction : restrictions) {
+            userConfig.add(new NamedValue(restriction.getValue(), Boolean.toString(false)));
```

Comment:
What does this currently look like in the DB? I'd imagine a prefix might be useful for long-term upgrades, etc.